ReferenceManual.py

```diff
@@ -0,0 +1,23 @@
+from ColorPair import get_total_major_colors as major_len
+from ColorPair import get_total_minor_colors as minor_len 
+import ColorPairGenerator as cpg
+
+def createReferenceData():

```

Comment:
Nice separation of reference manual. Try writing a test for `createReferenceData` and `printTable`

Reply:
One test case might be added of checking if the any new color code pair combination exists in reference manual or not
Below is a short addition of code line in file ColorCodeUnitTest.py added  
![referenceManualTestcase](https://user-images.githubusercontent.com/15873999/109269268-4702dd00-7832-11eb-9716-5ca7691c5e45.png)